jpa-crud/jpa-crud-java-sbt/jpa-crud-java-sbt-impl/src/main/resources/evolutions/default/1.sql

```diff
@@ -0,0 +1,18 @@
+# --- !Ups
+
+CREATE TABLE ENTITY (
+  ID CHAR(36) PRIMARY KEY,
+  PROPERTY VARCHAR(100 CHAR) NOT NULL,
+);
+
+CREATE TABLE ENTITY_PART (
+  ENTITY_ID CHAR(36),
+  NAME VARCHAR(100 CHAR),
+  PRIMARY KEY (ENTITY_ID, NAME),
+  CONSTRAINT ENTITY_PART_FK FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)
+);
+
+# --- !Downs
+
+DROP TABLE ENTITY_PART;
+DROP TABLE ENTITY;
```

Comment:
I don't think it's a good idea to have this file on the `evolutions/default` folder.

Evolutions don't work in Lagom (neither dev mode nor production) so I'd rather not mislead people into thinking otherwise. I think this file should be placed in `src/main/sql` and not even use the `--- !Ups` /`--- !Downs` syntax.

Reply:
Evolutions work fine in Lagom 😉We use Evolutions in production for our services. 
This recipe not use the setting `hibernate.hbm2ddl.auto`  in persistence.xml (`none` default value). Without Evolutions, this recipe would not be passed any test and would not be running 😄